tda/streaming.py

```diff
@@ -189,7 +190,10 @@ async def _init_from_principals(self, principals):
         # Initialize socket
         wss_url = 'wss://{}/ws'.format(
             principals['streamerInfo']['streamerSocketUrl'])
-        self._socket = await websockets.client.connect(wss_url)
+        if self._ssl_context:
```

Comment:
I believe you can pass the websocket context directly rather than doing this checking logic here. i.e., if you pass `None` I believe it should work.

Reply:
Is that try -- if the websockets folks set a default for ssl_context which is not None, wont it break? AFAICT looking at setup.py / requirements.txt we don't tightly pin deps.

I mean, I'm fine with that (it's less code), but that was my reasoning.

Reply:
I think it's safe to do this. Changing `ssl_context` to be a non-None default would be an API-facing change on their end, so let's cross that bridge when we come to it.